otter/integration/lib/cloud_load_balancer.py

```diff
@@ -264,3 +266,36 @@ def poll():
 """
 Matcher that asserts that all the given IPs are on the load balancer.
 """
+
+
+@attributes([
+    Attribute('pool', default_value=None),
+    Attribute('clb', instance_of=CloudLoadBalancer),
+    Attribute('rcs', instance_of=TestResources)
+])
+class CloudLoadBalancerController(object):
+    """
+    The CloudLoadBalancerController class provides an interface that allows
+    out-of-band control over a cloud load balancer through Mimic's API.
+    """
+
+    def enterPendingDeleteState(self):
```

Comment:
Can this be a more general "change_clb_state" function, that can set it to "PENDING_DELETE", "PENDING_UPDATE", or "ERROR" (e.g. it just takes a state to set it to)?